turbinia/evidence.py

```diff
@@ -965,6 +966,32 @@ def __init__(self, plaso_version=None, *args, **kwargs):
     self.copyable = True
     self.plaso_version = plaso_version
 
+  def validate(self):
+    cmd = [
+        'pinfo.py',
+        '--output-format',
+        'json',
+        '--sections',
+        'events',
+        self.local_path,
+    ]
+    total_file_events = 0
+
+    try:
+      log.info('Running pinfo.py to validate Plaso file')
```

Comment:
Small nit, but maybe include the name of the .plaso file in some of the log lines here (or at least in the exception string) to make it a little easier to see what file is affected.

Reply:
Oh, looks like this actually got added!  I'll resolve this comment.